files/styles/about.css

```diff
@@ -919,6 +919,12 @@ section {
     position: relative;
 }
 
+.web-dev-col{
+  display: flex;
+  justify-content: center;
+  text-align: center;
```

Comment:
Hey @jguadarr974 , everything looks solid. I would just remove the text-align line.